### PR TITLE
[Fix](statistics)Fix hive table statistic bug.

### PR DESCRIPTION
**Pull request by Jibing-Li**
Fix hive table statistic bug. Collect table/partition level statistics.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/statistics/BaseAnalysisTask.java [hunk 1]
```diff
@@ -142,6 +142,9 @@ private void init(AnalysisTaskInfo info) {
                     info, AnalysisState.FAILED,
                     String.format("Table with name %s not exists", info.tblName), System.currentTimeMillis());
         }
+        if (info.externalTableLevelTask) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/BaseAnalysisTask.java:145]: Add some comment for this `if`

File: fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStatistic.java [hunk 2]
```diff
@@ -138,10 +138,7 @@ public static ColumnStatistic fromResultRow(ResultRow resultRow) {
             String colName = resultRow.getColumnValue("col_id");
             Column col = StatisticsUtil.findColumn(catalogId, dbID, tblId, idxId, colName);
             if (col == null) {
-                LOG.warn("Failed to deserialize column statistics, ctlId: {} dbId: {}"
-                                + "tblId: {} column: {} not exists",
-                        catalogId, dbID, tblId, colName);
-                return ColumnStatistic.UNKNOWN;
+                return columnStatisticBuilder.build();
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStatistic.java:141]: Add some comment

**Comment by Jibing-Li** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**